app/controllers/application_controller.rb

```diff
@@ -20,14 +18,6 @@ def layout_by_resource
     devise_controller? ? 'admin' : 'application'
   end
 
-  def setup_browser_rules
-    Browser.modern_rules.clear
-    Browser.modern_rules << ->(b) { b.chrome? && b.version.to_i >= 55 }
-    Browser.modern_rules << ->(b) { b.firefox? && b.version.to_i >= 51 }
-    Browser.modern_rules << ->(b) { b.safari? && b.version.to_i >= 9 }
-    Browser.modern_rules << ->(b) { b.ie? && b.version.to_i >= 10 }
```

Comment:
Current versions of everygreen browsers

* Chrome = 80
* Firefox = 75
* Safari = 13

so I think we can safely ignore checking any of those. We still need to display a message to IE 9 or older - see the conditional HTML comment below

Reply:
Hey  - just checking you ran this past Micky? She had some good reasons to support older browser for some Deaf user and their families who have older browsers.

Reply:
This change doesn't actually change our browser support at all. It just changes **how** we present the "your browser is not supported" message.

Reply:
We agreed not to support IE legacy browsers - our Analytics showed that it was a very small user group in comparison with Chrome and Safari.